src/services/netMgr.js

```diff
@@ -0,0 +1,216 @@
+        route = '/' + route;
+    }
+    return this.axiosInstance.get(route)
+        .then(cb)
+        .catch(err || this.logAJAXErrors);
+};
+
+/**
+ * AXIOS post wrapper;
+ *
+ * @param route
+ * @param postData
+ * @param cb
+ * @param err
+ * @returns {Promise.<TResult>}
+ */
+
+// TODO: these are awefully similar functions... merege them?
+
+NetMgr.apiPost = function (route, postData, cb, err) {
+    if (!route.match(/^\//)) {
+        route = '/' + route;
+    }
+    return this.axiosInstance.post(route, postData)
+        .then(cb)
+        .catch(err || this.logAJAXErrors);
+};
+
+/**
+ * Default error handler, if one isn't provided to the api* ones.
+ * @param error
+ */
+
+NetMgr.logAJAXErrors = function (error) {
+    if (error.response) {
+        // non 2xx response
+        console.log(error.response);
+        console.log(error.response.status);
+        console.log(error.response.headers);
+    } else if (error.request) {
+        console.log(error.request);
+    } else {
+        // catchall
+        console.log('Error', error.message);
+    }
+    console.log(error.config);
+};
+
+/**
+ * Checks is Mocks are applied
+ *
+ * @returns {null}
+ */
+
+NetMgr.isMocked = function () {
+    // TODO: make tis more effective
+    return this.mocker;
+};
+
+/**
+ * Sets up or restores a Mock Adapter
+ */
+
+NetMgr.mockOn = function () {
+    // if there's a mockAdapter add it back in.
+    if (this.mockAdapter instanceof MockAdapter) {
+        this.axiosInstance.defaults.adapter = this.mockAdapter;
+        this.mocker = this.mockAdapter;
+    } else {
+        this.mocker = new MockAdapter(this.axiosInstance);
+        Fixtures.apply(this.mocker);
+    }
+};
+
+/**
+ * Removes and stores Mock Adapter and restores original Axios instance it hides
+ */
+
+NetMgr.mockOff = function () {
+    if (this.isMocked()) {
+        // save the current mock set
+        this.mockAdapter = this.axiosInstance.defaults.adapter;
+        // restore axios's previous adapter
+        this.mocker.restore();
+    }
+    this.mocker = false;
+};
+
+
+// FIRED on instantiation/
+// set cookie in devtools to ignore mocks in development and connect directly to local API
+// document.cookie = "arcv_ignore_mocks=true;max-age=" + 86400*30;
+if (Config.env === "development" && ( document.cookie.indexOf("arcv_ignore_mocks=true") === -1 )) {
+    NetMgr.mockOn();
+}
+
+// Add interceptor to detect an expired access_token and refresh;
+NetMgr.axiosInstance.interceptors.response.use(
+    function (response) {
+        // everything fine! return the response to apiGet/apiPost
+        return response;
+    },
+    function (error) {
+
+        // Get the original request configuration.
+        var origCfg = error.config;
+
+        // Get the error from that request.
+        var origResp = error.response;
+
+        // Is it a 401 we havn't seen before? (and do we have an old token set)
+        if (origResp.status === 401 && !origCfg._retry && this.token) {
```

Comment:
Ah! right, will comment "Unauthorized" out and change description.
Any idea why "Unauthenticated." has that full stop?